packages/itwin/grouping-mapping-widget/src/widget/components/OverlappedElementsInformationPanel.tsx

```diff
@@ -0,0 +1,153 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { useCallback, useMemo, useState } from "react";
+import type { CreateTypeFromInterface } from "../utils";
+import {
+  InformationPanel,
+  InformationPanelBody,
+  InformationPanelHeader,
+  Table,
+  Text,
+  toaster,
+} from "@itwin/itwinui-react";
+import "./OverlappedElementsInformationPanel.scss";
+import type { Group } from "@itwin/insights-client";
+import type { OverlappedInfo } from "./context/GroupHilitedElementsContext";
+import type { CellProps } from "react-table";
+import { useGroupHilitedElementsContext } from "./context/GroupHilitedElementsContext";
+import { clearEmphasizedOverriddenElements, clearHiddenElements, visualizeElements, zoomToElements } from "./viewerUtils";
+
+export interface OverlappedElementsInformationPanelProps {
+  group?: Group;
+  onClose: () => void;
+  overlappedElementsInfo: Map<string, OverlappedInfo[]>;
+  groups: Group[];
+}
+
+export interface OverlappedElementsDisplayProps {
+  numberOfElements: string;
+  groups: string[];
+  elementsIds: string[];
+}
+type OverlappedTyped = CreateTypeFromInterface<OverlappedElementsDisplayProps>;
+
+export const OverlappedElementsInformationPanel = ({
+  group,
+  onClose,
+  overlappedElementsInfo,
+  groups,
+}: OverlappedElementsInformationPanelProps) => {
+  const [isOverlappedInfoLoading, setIsOverlappedInfoLoading] =
+    useState<boolean>(false);
+  const { setIsOverlappedColored } = useGroupHilitedElementsContext();
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const columns = useMemo(
+    () => [
+      {
+        Header: "Table",
```

Comment:
Where is this informative table header displayed?

Reply:
No where. Old iTwinUI requirement.